app/controllers/teams_controller.rb

```diff
@@ -0,0 +1,114 @@
+class TeamsController < ApplicationController
+  load_and_authorize_resource
+  def index
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create
+    compnay = Company.first
+    @team.name = params[:team][:name]
+    @team.company = compnay
+    respond_to do |format|
+      if @team.save
+        flash[:success] = 'Team Created Successfully'
+      else
+        flash[:danger] = 'An Error Occured'
+      end
+      redirect_to teams_path
+      format.js
+    end
+  end
+
+  def edit
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def update
+    @team.update(team_params)
+    respond_to do |format|
+      flash[:success] = 'Team Updated Successfully'
+      redirect_to teams_path
+      format.js
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def destroy
+    @team.destroy
+    respond_to do |format|
+      flash[:danger] = 'Team Destroyed Successfully'
+      redirect_to teams_path
+      format.html
+    end
+  end
+
+  def new_member
+    respond_to do |format|
+      format.js
+    end
+  end
+
+  def create_member
+    company = Company.first
+    user = User.find(params[:team][:user])
+    result = TeamUser.create(team: @team, user: user, company: company)
+    respond_to do |format|
+      if result
+        flash[:success] = 'User Added Successfully'
+      else
+        flash[:danger] = 'An Error Occured'
+      end
+      redirect_to team_path(@team)
+      format.js
+    end
+  end
+
+  def remove_member
```

Comment:
```suggestion
  def remove_team_user
```

or remove_member_from_team